bindings/swift/Cargo.toml

```diff
@@ -9,8 +9,10 @@ edition = "2021"
 crate-type = ["cdylib", "staticlib"]      # Creates dynamic lib
 
 [dependencies]
+fern-logger = { version = "0.5.0", default-features = false }
 futures =  { version = "0.3.23", default-features = false }
-iota-wallet = { path = "../../", version = "0.2.0", features = ["message_interface", "events"] }
+iota-wallet = { path = "../../", features = ["message_interface", "events"] }
```

Comment:
```suggestion
iota-wallet = { path = "../../", default-features = false, features = [ "message_interface", "events" ] }
```

Reply:
we should add features `"mnemonic", "storage", "stronghold"` to be consistent with how it was

Reply:
My bad, I edited. I just really wanted the spaces :)